recipe/meta.yaml

```diff
@@ -1,30 +1,26 @@
-{% set version = "7.4.2" %}
+{% set version = "8.9.0" %}
 
 package:
   name: gradle
   version: {{ version }}
 
 source:
   url: https://github.com/gradle/gradle/archive/v{{ version }}.tar.gz
-  sha256: 4cbfde441a8afd96a8503a120402839838e3d6bad95aed5d015e642f9c40668c
+  sha256: 9172bbdc47265e35ced086c48a56b39d52376e1e1b4417d38583b4e4b25f6d2f
 
 build:
   number: 0
+  noarch: generic
 
 requirements:
-  build:
-    - {{ compiler('c') }}
-    - sysroot_linux-64 2.17  # [linux64]
   host:
-    - openjdk >=11
+    - openjdk >=11,<12
 
 test:
   requires:
-    - openjdk >=11
-    - m2-grep  # [win]
+    - openjdk >=8,<23
   commands:
     - gradle -v
-    - gradle -v | grep -E "{{ version.replace(".", "\.") }}" || exit 1
```

Comment:
I removed this test because it doesn't work with `.0` versions. gradle just prints `8.9` instead of `8.9.0`.

Reply:
We can do something like
```yaml
    - gradle -v | grep -E "{{ "\.".join(version.split(".")[:2]) }}" || exit 1
```
or 
```yaml
    # gradle doesn't show patch version if it's 0
    {% set maj_min = "\.".join(version.split(".")[:2]) %}
    - gradle -v | grep -E "{{ maj_min }}" || exit 1
```

Reply:
Done